examples/stable/test/property/canister_methods/on_low_wasm_memory/test/tests.ts

```diff
@@ -0,0 +1,176 @@
+import { Principal } from 'azle';
```

Comment:
Jordan deeply scrutinize once all comments are addressed, this will be the final comment